lib/public/views/Runs/format/formatRunDetectors.js

```diff
@@ -29,48 +29,56 @@ export const formatRunDetectors = (run, detailsModel) => {
         return '-';
     }
 
+    /**
+     * Creates a component containing the detector badges
+     *
+     * @param {List<Detector>} detectors list of detectors
```

Comment:
The syntax for arrray is `Detector[]` (prefered) or `Array<Detector>`
However the type you pass is not a `Detector` array, but an array of detector qualities. Could you please create a typedef file in `lib/domain/entities`, and add in it the typedef:
```js
/**
 * @typedef RunDetectorQuality
 * @property {number} id
 * @property {string} name
 * @property {string[]} quality
 */
```

Then you can use it to type this param. (Please also use this type in `lib/domain/entities/Run` to replace line 67 property type)